Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1854,6 +1854,14 @@
     SYS_umask
     SYS_work_interval_ctl))
 
+;; <rdar://problem/119946688>
```

Comment:
Not sure this line is needed given we have the radar number in the commit message.